lib/Cro/WebApp/Template/AST.rakumod

```diff
@@ -15,8 +15,10 @@ my role ContainerNode does Node {
 my class Template does ContainerNode is export {
     has @.used-files;
 
-    method compile() {
+    method compile(:$no-fragment) {
         my $*IN-SUB = False;
+        my $*IN-FRAGMENT = False;
+        my $*NO-FRAGMENT = $no-fragment;
```

Comment:
Is this dead code or missing the implementation?

Reply:
good catch - this is indeed dead code - compile now just makes both the fragment and the whole - and then the render phase picks out the fragment if needed at method render (Repository.rakumod ll31-38)